decidim-core/app/cells/decidim/content_blocks/footer_sub_hero/show.erb

```diff
@@ -5,7 +5,7 @@
     </h2>
   </div>
   <p class="text-lg text-gray-2">
-    <%== t("decidim.pages.home.footer_sub_hero.footer_sub_hero_body") %>
+    <%== t("decidim.pages.home.footer_sub_hero.footer_sub_hero_body_html") %>
```

Comment:
I have made that already but discarded it, as when I saw the final result I realized that we were already using the description in another homepage content block: "Sub hero banner".

![Screenshot of the "Sub hero banner" content block](https://github.com/decidim/decidim/assets/717367/ceaecdf4-9ee5-4605-a426-a87600d523cf)

So, having the same text two times in the homepage (three if we count the general page footer that we're adding in this PR) it's too much IMO.

Reply:
Yes, you are right, it may be a bit too much. also is the lead text for the register button.

Maybe is worth trying to change the footer content block to support an additional text, so that we enable people to customize the text ?

Reply:
I agree. There's already an approved proposal for that: https://meta.decidim.org/processes/roadmap/f/122/proposals/15693?commentId=23765